options/locale/locale_en-US.ini

```diff
@@ -3071,6 +3071,10 @@ empty.repo = Did you upload a package, but it's not shown here? Go to <a href="%
 filter.type = Type
 filter.type.all = All
 filter.no_result = Your filter produced no results.
+filter.version.asc = A - Z
+filter.version.desc = Z - A
+filter.created.asc = Oldest
+filter.created.desc = Newest
```

Comment:
```suggestion
filter.version.asc = Ascending
filter.version.desc = Descending
filter.created.asc = Oldest First
filter.created.desc = Newest First
```

Reply:
Also I'd move these to root options namespace, they are universally useful.

Reply:
I used the same as Docker Hub (see #21430). To be honest, it must be named `Ascending Version` because in another list which shows the package name too, you could not distinguish between "Package Name Ascending" and "Version Ascending". But the changed page just shows versions. So I could not decide how they should be named.

Reply:
I see, so it's relying on the version never becoming alphabetical, otherwise it would be ambigous. I guess you can keep the A-Z, but I still prefer the "First" suffix on the date ones and moving them to translation root.

Reply:
No, I'm also in favor of `First`. I'm always confused by GitHub…
```suggestion
filter.version.asc = A - Z
filter.version.desc = Z - A
filter.created.asc = Oldest first
filter.created.desc = Newest first
```